nni/tools/nnictl/launcher.py

```diff
@@ -378,6 +403,20 @@ def set_experiment(experiment_config, mode, port, config_file_name):
             {'key': 'aml_config', 'value': experiment_config['amlConfig']})
         request_data['clusterMetaData'].append(
             {'key': 'trial_config', 'value': experiment_config['trial']})
+    elif experiment_config['trainingServicePlatform'] == 'heterogeneous':
+        request_data['clusterMetaData'].append(
+            {'key': 'heterogeneous_config', 'value': experiment_config['heterogeneousConfig']})
+        platform_list = experiment_config['heterogeneousConfig']['trainingServicePlatforms'].split(',')
+        for platform in platform_list:
+            if platform === 'aml':
```

Comment:
It looks can be defined in a map from key to a couple of strings.

Reply:
fixed